health_check/management/commands/health_check.py

```diff
@@ -0,0 +1,28 @@
+import sys
+
+from django.core.management.base import BaseCommand
+
+from health_check.mixins import CheckMixin
+
+
+class Command(CheckMixin, BaseCommand):
+    help = "Run health checks and exit 0 if everything went well."
+
+    def handle(self, *args, **options):
+        # perform all checks
+        errors = self.errors
+
+        # use stdout io module for testing or sys.stdout by default
+        stdout = options.get('stdout', sys.stdout)
```

Comment:
No need to get it explicitly from the options, Django already does that for you. `sys.stdout` or whatever you pass via `call_command` is accessible via, `self.stdout`.
```suggestion
```